src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -108,21 +116,23 @@ function AboutPage(props) {
                                     <Text style={[styles.baseFontStyle, styles.mv5]}>{props.translate('initialSettingsPage.aboutPage.description')}</Text>
                                 </View>
                             </View>
-                            {_.map(menuItems, (item) => (
-                                <MenuItem
-                                    key={item.translationKey}
-                                    title={props.translate(item.translationKey)}
-                                    icon={item.icon}
-                                    iconRight={item.iconRight}
-                                    onPress={() => item.action()}
-                                    shouldBlockSelection={Boolean(item.link)}
-                                    onSecondaryInteraction={
-                                        !_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor) : undefined
-                                    }
-                                    ref={(el) => (popoverAnchor = el)}
-                                    shouldShowRightIcon
-                                />
-                            ))}
+                            <MenuItemList
+                                menuItems={_.map(menuItems, (item) => ({
+                                    key: item.translationKey,
+                                    title: props.translate(item.translationKey),
+                                    icon: item.icon,
+                                    iconRight: item.iconRight,
+                                    disabled: props.isShortcutsModalOpen,
+                                    onPress: item.action,
+                                    shouldShowRightIcon: true,
+                                    onSecondaryInteraction: !_.isEmpty(item.link)
+                                        ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor)
+                                        : undefined,
+                                    ref: (el) => (popoverAnchor = el),
+                                    shouldBlockSelection: Boolean(item.link),
+                                }))}
```

Comment:
Move `menuItems` mapping to separate `useMemo` variable. There is no need to recalculate it every rerender